next.config.js

```diff
@@ -15,12 +15,14 @@ module.exports = {
       process.env.DYNAMIC_LINK_DOMAIN || "https://dev.helio.social",
     WIX_URL: process.env.WIX_URL || 'https://home.helio.social/',
     LOCAL_STORAGE_HANDLER: `${process.env.HUMAN_ID_WEB_APP_REDIRECT_URL}/localstorage-handler` || 'https://www.helio.social/localstorage-handler',
-    HELIO_SECURE_URL: `${process.env.HUMAN_ID_WEB_APP_REDIRECT_URL}`
+    HELIO_SECURE_URL: `${process.env.HUMAN_ID_WEB_APP_REDIRECT_URL}`,
+    SEGMENT_WRITE_KEY: process.env.SEGMENT_WRITE_KEY,
   },
   serverRuntimeConfig: {
     HUMAN_ID_GET_WEB_LOGIN_URL: process.env.HUMAN_ID_GET_WEB_LOGIN_URL,
     HUMAN_ID_CLIENT_ID: process.env.HUMAN_ID_CLIENT_ID,
     HUMAN_ID_CLIENT_SECRET: process.env.HUMAN_ID_CLIENT_SECRET,
+    SEGMENT_WRITE_KEY: process.env.SEGMENT_WRITE_KEY,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `SEGMENT_WRITE_KEY` is added to both the public and server runtime configurations. This could potentially expose sensitive data to the client side, which can be a security risk if the key is not meant to be publicly accessible. If the key is only used on the server side, it should only be in the `serverRuntimeConfig`. If it's used on both sides, consider whether this is safe and necessary.
```diff
   serverRuntimeConfig: {
     HUMAN_ID_GET_WEB_LOGIN_URL: process.env.HUMAN_ID_GET_WEB_LOGIN_URL,
     HUMAN_ID_CLIENT_ID: process.env.HUMAN_ID_CLIENT_ID,
     HUMAN_ID_CLIENT_SECRET: process.env.HUMAN_ID_CLIENT_SECRET,
- 25:     SEGMENT_WRITE_KEY: process.env.SEGMENT_WRITE_KEY,
   },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->